redis_close

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Redis Close Script
+
```

Comment:
And we do not support Windows users running this script because the native Redis command-line tools don't work on Windows. Could you please add 'Supported OS: macOS and Linux' here? Thanks